pkg/controller/provider/container/vsphere/watch.go

```diff
@@ -161,6 +160,7 @@ func (r *VMEventHandler) run() {
 	defer r.log.Info("Run stopped.")
 	interval := time.Second * time.Duration(
 		Settings.PolicyAgent.SearchInterval)
+	r.list()
```

Comment:
The watch is setup with snapshot=false.  List and validate all of the VMs.  Then, go into the scheduled follow up searches.